python/ray/tests/test_dashboard.py

```diff
@@ -26,10 +26,14 @@ def test_ray_start_default_port_conflict(call_ray_stop_only, shutdown_only):
                 "--head",
                 "--port",
                 "9999",  # use a different gcs port
+                "--include-dashboard=True"
```

Comment:
@simon-mo Is it okay that I added these? Without this the error is not raised.

Reply:
yes!